frontend/package.json

```diff
@@ -26,19 +26,20 @@
     "marked": "^2.0.0",
     "material-ui-chip-input": "^1.1.0",
     "query-string": "^6.13.1",
+    "rc-slider": "^9.7.1",
     "react": "^16.13.1",
+    "react-avatar-editor": "^11.1.0",
     "react-bootstrap": "^0.33.1",
     "react-circular-progressbar": "^2.0.3",
     "react-cookie": "^4.0.3",
     "react-copy-to-clipboard": "^5.0.2",
     "react-dom": "^16.13.1",
+    "react-meta-tags": "^1.0.1",
```

Comment:
Can we use `react-helmet` instead? It's a more widely used library / more of a standard for adding meta tags or tags in the head.

Reply:
I have switched to use react-helmet. Thanks for the feedback!